lib/diagnostic.rb

```diff
@@ -91,13 +97,15 @@
 # In a Ruby string, write how you would remove the last two elements from this
 # array
 
-Diagnostic.remove_from_array = 'your answer here'
+Diagnostic.remove_from_array = '.delete_at(2, 3)'
```

Comment:
This method needs to be invoked on `arr` and `delete_at` only takes one argument.